lib/utils.pm

```diff
@@ -1454,6 +1455,20 @@ sub setup_static_network {
     assert_script_run "ping -c 1 10.0.2.2 || journalctl -b --no-pager > /dev/$serialdev";
 }
 
+=head2  is_arrays_differ
+ Comparing two arrays passed by reference. Return 1 if arrays has symmetric difference
+ and 0 otherwise.
+=cut
+sub is_arrays_differ {
```

Comment:
@foursixnine what do you think ?

Reply:
better english would be just `arrays_differ`

Reply:
and better than english is when boolean functions start from "is_" :)

Reply:
uhm, nope.

Reply:
@asmorodskyi sorry I didn't see the mention you did :), anywho. `if(arrays_differ){}` is way better than `if(is_arrays_differ)`.